enos/enos-scenario-package-manager.hcl

```diff
@@ -0,0 +1,217 @@
+scenario "package_manager" {
```

Comment:
Why does this need to be a different scenario? Feels like this could be a matrix variant in the existing scenarios. The reason it was a different scenario in the smoke tests was because of the difficulty of writing one module to handle so many different inputs. Here it feels like the only things are might be different are the package type, distro, and whether or not we want to manage the service. Is maintaining this as a different scenario more work than handling packages as a variant in the others?

Reply:
This discussion ended up in Slack but we agreed that this was a good idea, and I've gone ahead and integrated the package testing capability as a matrix variant in the other scenarios rather than a standalone scenario.